daemon/delete.go

```diff
@@ -141,9 +142,11 @@ func (daemon *Daemon) commonRm(container *Container, forceRemove bool) (err erro
 		return fmt.Errorf("Driver %s failed to remove root filesystem %s: %s", daemon.driver, container.ID, err)
 	}
 
-	initID := fmt.Sprintf("%s-init", container.ID)
-	if err := daemon.driver.Remove(initID); err != nil {
-		return fmt.Errorf("Driver %s failed to remove init filesystem %s: %s", daemon.driver, initID, err)
+	if runtime.GOOS != "windows" {
+		initID := fmt.Sprintf("%s-init", container.ID)
+		if err := daemon.driver.Remove(initID); err != nil {
```

Comment:
@swernli for such cases I suggest using null object pattern (or in this case nop function pattern) with Go build tags that  can provide compile-time safety. it would be the best if we can reduce inline `runtime.GOOS` comparisons to minimum.